### PR TITLE
added tomorrow night italics color scheme

### PR DESCRIPTION
**Pull request by kennethreitz**
- https://github.com/kennethreitz/sublime-tomorrow-night-italics-theme
- https://github.com/kennethreitz/sublime-tomorrow-night-italics-theme/releases

✨🍰✨

### CODE REVIEW

**Comment by kennethreitz** [comment]
This is an updated version (very late, sorry) of #5251

Thank you very much! Let me know if anything else needs to be updated. This will make my life easier :)

File: repository/t.json [hunk 1]
```diff
@@ -2175,6 +2175,17 @@
 			]
 		},
 		{
+			"name": "Tomorrow Night Italics Color Scheme",
+			"details": "https://github.com/kennethreitz/sublime-tomorrow-night-italics-theme",
+			"labels": ["color scheme"],
+			"releases": [
+				{
+					"sublime_text": "*",
+					"tags": "true"
```
Comment (FichteFoll) [inline repository/t.json]: This should be a boolean.

**Comment by kennethreitz** [comment]
damnit, wish this was easy to change now without recreating

**Comment by kennethreitz** [comment]
updated!

**Comment by FichteFoll** [comment]
>damnit, wish this was easy to change now without recreating

Recreating what?

**Pull request merged**